pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -17,7 +17,7 @@
     TestCase,
 )
 from pytest_embedded.utils import UserHint, remove_asci_color_code, to_str
-from pytest_embedded_serial.dut import SerialDut
+from pytest_embedded_serial.dut import Dut, SerialDut
```

Comment:
should be `from pytest_embedded.dut import Dut`

Reply:
now LinuxDut does not require install `pytest_embedded_serial`, would raise `ImportError` if you implement like this.

When you're fixing it, could you add a test as well?

Reply:
Yes, i've added this into the existing test